parser-service/src/test/resources/query-validation-consistency/api.raml

```diff
@@ -91,6 +91,11 @@ types:
     type: booleanItems | integerItems
   nonNullableUnionOfNullableArrays:
     type: nullableStringItems | nullableIntegerItems
+  repeatable:
+    type: integerItems | booleanItems | nil
+  notRepeatable:
+    type: integerItems | nullableString #Since integerItems is an array we should consider this repeatable, we keep
```

Comment:
I would get rid of this comment

Reply:
I also had doubts about it @JuanAller asked me about it, what do you think Juan ?

Reply:
I am ok with the user-story, maybe we can point the user story to this commit as a help for the future change